packages/checkbox-group/stories/index.stories.mdx

```diff
@@ -62,20 +62,17 @@ You can pre-select options by targeting the `modelValue` object of the option an
 
 <Story name="Pre-select">
   {html`
-    <lion-checkbox-group name="scientistsGroup" label="Favorite scientists">
+    <lion-checkbox-group name="scientists" label="Favorite scientists">
```

Comment:
shouldn't that be `scientists[]`?

we should throw an error if you try to use a name without an ending `[]` for a checkbox-group

Reply:
Yes by convention it should. To my knowledge it was not spec'd that we should throw an error if people don't use this convention for multiple choice though.